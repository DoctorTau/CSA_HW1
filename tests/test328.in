409
5201 -4981 4950 -5230 7575 1136 -5420 1676 -2838 -1840 3354 787 -3746 2720 1267 -9657 -8731 -6651 8217 6900 -4471 4061 6754 9534 -1407 4326 1868 -6785 5505 -6922 5326 -6375 1069 -5765 1452 4864 -6292 -4630 -2924 -538 3085 -8971 -5179 -7543 1731 3466 -828 5557 3817 3181 -6599 -8556 3824 -4518 -672 -6445 6230 6042 -7261 7275 -6006 -701 -7338 6677 2409 7560 7217 -6285 7373 1352 -1566 -9292 -3715 7278 -7343 1143 -4179 4482 8757 7316 -9404 -393 -129 6085 6198 -3750 7286 3431 -6649 -1377 3324 535 1878 8028 9439 8451 -3560 8684 -2573 -9475 -6057 -9571 6741 -1590 -3958 8280 644 8062 9571 -4378 -9434 -2885 9818 9191 -7071 -3125 -7112 7582 -3243 -3576 313 9662 5219 2971 -9553 8121 -7874 8305 8105 5297 -6293 -7028 4928 7476 -5952 -4694 -7086 1391 -9056 -1765 -3618 2674 -7053 -1133 28 6327 5588 1069 -6124 7187 -8624 4081 -9314 -6338 91 2939 7089 3680 4314 -2096 -7222 6612 -431 -8970 132 -582 -3854 2314 -1164 5165 6213 6285 -6803 -4894 1490 -4940 2225 -9084 -364 -4816 -4490 7222 -4447 -4758 -4742 -6772 -9743 7924 4013 -6470 -6170 2722 -9118 -7273 4026 -9554 -4958 -5184 -397 1189 2954 -7909 -8728 -2407 7774 2597 -4535 4418 -5275 8044 1728 1113 3578 7875 -1242 -7451 -9803 -6142 4568 -9246 -3993 8005 5736 -8035 610 3838 9169 291 -9497 -8883 -6830 -6630 8557 787 -8594 4989 -3019 -9048 -700 9412 -453 -1608 -5009 819 3193 -779 9555 -6389 -9651 8290 7116 -7724 7613 8788 9218 -564 8153 4351 2981 -322 -9667 5529 6851 4035 -1289 -7797 2966 -5600 -2494 7958 -738 8152 3152 3559 -2917 -7398 3312 6603 -7922 -1178 -5788 -4443 -6554 -885 8163 -2182 -1434 -458 5779 6434 -8077 7668 -6272 -1585 3900 2660 -950 -3519 -7157 1556 1733 1810 -4716 -4103 -7679 -5084 6856 6730 1955 -4494 -1751 -3315 5876 8035 6501 6468 6100 -8633 7515 9110 -5050 -1896 -7872 6455 7112 -7045 -6782 7751 1715 4169 6679 4818 -4440 -9332 714 2394 -3797 1449 -2309 -4903 7836 -6191 294 6725 -323 -5714 -6094 -3916 8608 3746 -1912 9327 -5364 -3409 3593 9245 1154 -4255 1583 3637 -16 9638 4330 4552 4478 859 -6006 -2096 -3368 6628 2566 5434 -7255 -9156 -9477 -7393 -5672 -8940 7333 -2537 -3155 726 9939 9857 1351 3485 -8128 3476 -2606 -1340 7862 -8632 -8441 -8337 5366 9402 6884 9262 118 -7269 2248 735 -2338 -6522 -8102 -9804 -5259 5731 6682 