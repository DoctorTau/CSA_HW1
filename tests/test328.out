-5230 -5420 -2838 -1840 -3746 -6922 -6292 -4630 -2924 -538 -828 -8556 -4518 -672 -6006 -7338 -1566 -9292 -9404 -3750 -3560 -1590 -3958 -4378 -9434 -7112 -3576 -7874 -7028 -5952 -4694 -7086 -9056 -3618 -6124 -8624 -9314 -6338 -2096 -7222 -8970 -582 -3854 -1164 -4894 -4940 -9084 -364 -4816 -4490 -4758 -4742 -6772 -6470 -6170 -9118 -9554 -4958 -5184 -8728 -1242 -6142 -9246 -6830 -6630 -8594 -9048 -700 -1608 -7724 -564 -322 -5600 -2494 -738 -7398 -7922 -1178 -5788 -6554 -2182 -1434 -458 -6272 -950 -4716 -5084 -4494 -5050 -1896 -7872 -6782 -4440 -9332 -5714 -6094 -3916 -1912 -5364 -16 -6006 -2096 -3368 -9156 -5672 -8940 -8128 -2606 -1340 -8632 -2338 -6522 -8102 -9804 